bridges/otelzap/README.md

```diff
@@ -0,0 +1,5 @@
+# OpenTelemetry Zap Log Bridge
+
+[![Go Reference](https://pkg.go.dev/badge/go.opentelemetry.io/contrib/bridges/otelzap.svg)](https://pkg.go.dev/go.opentelemetry.io/contrib/bridges/otelzap)
+
+This package implements a [Log Bridge](https://opentelemetry.io/docs/specs/otel/logs/bridge-api/) for the [zap](https://github.com/uber-go/zap) logger. This bridge can be used to send logs emitted by application using zap to an OpenTelemetry compatible endpoint through the [OpenTelemetry Logs Bridge API](https://pkg.go.dev/go.opentelemetry.io/otel/log).
```

Comment:
```suggestion
This package implements a [Log Bridge](https://opentelemetry.io/docs/specs/otel/logs/bridge-api/) for [zap](https://github.com/uber-go/zap) logger. This bridge can be used to send logs emitted by application using zap to an OpenTelemetry compatible endpoint through the [OpenTelemetry Logs Bridge API](https://pkg.go.dev/go.opentelemetry.io/otel/log).
```

Zap is usually referred as just zap. It's not The Batman 😅

Reply:
Sure, fixed :)